From 21d6fcb4099ce2a88ed5ebdc35987a03c14828a2 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 2 Dec 2010 20:55:42 +0800
Subject: [PATCH] ENGR00133052 MX53: Fix system hang in vpu decoding with iram.

Change vpu clock dependency for iram to emi_intr_clk[0]
since vpu should use emi_intr_clk[0], not emi_intr_clk[1].

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/mach-mx5/clock.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx5/clock.c b/arch/arm/mach-mx5/clock.c
index 783d973..606471e 100644
--- a/arch/arm/mach-mx5/clock.c
+++ b/arch/arm/mach-mx5/clock.c
@@ -4755,9 +4755,6 @@ int __init mx53_clocks_init(unsigned long ckil, unsigned long osc, unsigned long
 	esdhc3_clk[0].get_rate = _clk_esdhc3_get_rate;
 	esdhc3_clk[0].set_rate = _clk_sdhc3_set_rate;
 
-#ifdef CONFIG_MXC_VPU_IRAM
-	 vpu_clk[2].secondary = &emi_intr_clk[1];
-#endif
 #if defined(CONFIG_USB_STATIC_IRAM) \
     || defined(CONFIG_USB_STATIC_IRAM_PPH)
 	usboh3_clk[1].secondary = &emi_intr_clk[1];
-- 
1.5.4.4

